### PR TITLE
#2163 more french translations

### LINKED ISSUE #2163: More French translations
## Describe the bug

Require a few more french translations!

### To reproduce

Dispensing development

### Expected behaviour

Dispensing development

### Proposed Solution

Dispensing development

### Version and device info

Dispensing development

### Additional context

Dispensing development

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #2163 

## Change summary

- Adds some translations that I googled
- Actually sets the language of the new translation files - oopsies
- Uses localization strings in the last few places (maybe?)

## Testing

N/A

### Related areas to think about

N/A

### CODE REVIEW

**Comment by craigdrown** [comment]
@alainsussol  can you check these? Thanks.

File: src/localization/dispensingStrings.json [hunk 1]
```diff
@@ -74,10 +76,15 @@
     "patients": "Patients",
     "payment_amount": "Montant du paiement",
     "payment": "Paiement",
+    "payments": "Paiments",
+    "receipts": "Reçus",
     "prescriber": "Prescripteur",
     "prescribers": "Prescripteurs",
     "select_a_policy": "Sélectionnez une police d'assurance",
     "select_items": "Sélectionnez des articles",
+    "select_a_payment_type": "sélectionnez un type de paiement",
```
Comment (mark-prins) [inline src/localization/dispensingStrings.json]:
```suggestion
    "select_a_payment_type": "Sélectionnez un type de paiement",
```

and UPPERCASE first letter for the next two too?

File: src/localization/formInputStrings.json [hunk 2]
```diff
@@ -27,7 +27,8 @@
     "policy_type": "Policy type",
     "registration_code": "Registration code",
     "unique_personal_policy": "Must be a unique personal policy number between 0 and 50 characters.",
-    "yes": "yes"
+    "yes": "yes",
```
Comment (mark-prins) [inline src/localization/formInputStrings.json]: 🤔 should your "yes" be "Yes"?
The "Oui" is capitalised.

File: src/widgets/Tabs/ItemSelect.js [hunk 3]
```diff
@@ -88,8 +88,8 @@ const ItemSelectComponent = ({
           <PrescriptionCart isDisabled={isComplete} />
 
           <FlexRow justifyContent="flex-end">
-            <PageButton text="Cancel" onPress={onDelete} style={{ marginRight: 7 }} />
-            <PageButton text="Next" onPress={canProceed ? nextTab : showToast} />
+            <PageButton text={buttonStrings.cancel} onPress={onDelete} style={{ marginRight: 7 }} />
+            <PageButton text={buttonStrings.confirm} onPress={canProceed ? nextTab : showToast} />
```
Comment (mark-prins) [inline src/widgets/Tabs/ItemSelect.js]: Not *Next*?

File: src/widgets/Tabs/PrescriberSelect.js [hunk 4]
```diff
@@ -108,12 +112,12 @@ const PrescriberSelectComponent = ({
...
       <FlexRow justifyContent="flex-end" alignItems="flex-end">
         <PageButton
-          text="Cancel"
+          text={buttonStrings.cancel}
           onPress={() => onCancelPrescription()}
           style={{ marginRight: 7 }}
         />
         <PageButton
-          text="OK"
+          text={buttonStrings.confirm}
```
Comment (mark-prins) [inline src/widgets/Tabs/PrescriberSelect.js]: *Next* / *Suivant*?

xliff: 17008,210

File: src/localization/formInputStrings.json (hunk 2 above)
Comment (joshxg) [reply src/localization/formInputStrings.json]: c05de31

File: src/localization/dispensingStrings.json (hunk 1 above)
Comment (joshxg) [reply src/localization/dispensingStrings.json]: 6f89e6d

File: src/widgets/Tabs/PrescriberSelect.js (hunk 4 above)
Comment (joshxg) [reply src/widgets/Tabs/PrescriberSelect.js]: bd1ad37

**Pull request merged**